apps/shell/src/app/modules/settings/pages/settings-general/github-conenction/github-connection.tsx

```diff
@@ -60,10 +67,16 @@ export function GithubConnection() {
           </TextBlock>
 
           <form
-            action={`https://github.com/organizations/${organization}/settings/apps/new?state=1234`}
+            action={`https://github.com/organizations/${organization}/settings/apps/new`}
             method="POST"
             data-testid="settings-github-connection__form"
           >
+            <input
+              className="hidden"
+              name="state"
+              readOnly
+              value={`${formState}`}
```

Comment:
Just a string so it should be safe to directly reference.

```suggestion
              value={formState}
```